stdlib/public/core/swiftCore.1.manifest.in

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes'?>
```

Comment:
Is this file being used by the old build at all? If it isn't, we can track it under the `Runtimes/Core` directory.

Reply:
I think that depends on how quickly we can migrate fully to the new build system. Most of the work will happen in the runtimes build because it is significantly easier to work with. But the goal is to backport this into the old build system if we cannot get a complete build from the new build system.